README.md

```diff
@@ -64,12 +62,44 @@ jobs:
           prefect-workspace: ${{ secrets.PREFECT_WORKSPACE }}
 
       - name: Run Prefect Deploy
-        uses: PrefectHQ/actions-prefect-deploy@v2
+        uses: PrefectHQ/actions-prefect-deploy@v3
+        with:
+          deployment-names: Simple
+          requirements-file-paths: ./examples/simple/requirements.txt
+```
+
+### Multi-Deployment Prefect Deploy
+
+Deploy multiple Prefect deployments that doesn't have a `push` step defined in the `prefect.yaml`
+```yaml
+name: Deploy multiple Prefect deployments
+on:
+  push:
+    branches:
+      - main
+jobs:
+  deploy_flow:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: checkout@v3
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+
+      - name: Prefect Auth
+        uses: PrefectHQ/actions-prefect-auth@v1
+        with:
+          prefect-api-key: ${{ secrets.PREFECT_API_KEY }}
+          prefect-workspace: ${{ secrets.PREFECT_WORKSPACE }}
```

Comment:
maybe just `...` these steps if they're the same as the single deployment case?

Reply:
nice good point